### PR TITLE
Add deployment script for release 2.0

### PR DESCRIPTION
**Pull request by elenadimitrova**
Updates all modules, i.e. `ApprovedTransfer`,  `CompoundManager`, `GuardianManager`, `LockManager`, `NftTransfer`, `RecoveryManager`, `TokenExchanger`, `MakerV2Manager` and `TransferManager`.

Infrastructure contracts also updated are: `BaseWallet`, `WalletFactory` and `TokenPriceProvider`.

### CODE REVIEW

File: deployment/7_upgrade_2_0.js [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
+  "ApprovedTransfer",
+  "CompoundManager",
+  "GuardianManager",
+  "LockManager",
+  "NftTransfer",
+  "RecoveryManager",
+  "TokenExchanger",
+  "MakerV2Manager",
+  "TransferManager"];
+const MODULES_TO_DISABLE = [];
```
Comment (juniset) [inline deployment/7_upgrade_2_0.js:37]: I think we need to disable MakerManager.

File: deployment/7_upgrade_2_0.js [hunk 2]
```diff
@@ -49,26 +66,23 @@ const deploy = async (network) => {
...
-  const changeMakerRegistryOwnerTx = await MakerRegistryWrapper.contract.changeOwner(config.contracts.MultiSigWallet, { gasPrice });
-  await MakerRegistryWrapper.verboseWaitForTransaction(changeMakerRegistryOwnerTx, "Set the MultiSig as the owner of the MakerRegistry");
-  const TokenPriceProviderWrapper = await deployer.wrapDeployedContract(TokenPriceProvider, config.contracts.TokenPriceProvider);
+  // Deploy the Base Wallet Library
+  const BaseWalletWrapper = await deployer.deploy(BaseWallet);
+  // Deploy TokenPriceProvider
+  const TokenPriceProviderWrapper = await deployer.deploy(TokenPriceProvider);
+  // Deploy the Wallet Factory
+  const WalletFactoryWrapper = await deployer.deploy(WalletFactory, {},
+    ModuleRegistryWrapper.contractAddress, BaseWalletWrapper.contractAddress, ENSManagerWrapper.contractAddress);
```
Comment (juniset) [inline deployment/7_upgrade_2_0.js:81]: If we deploy a new Factory we need to add it as a Manager of the ENSManager.

Comment (elenadimitrova) [reply deployment/7_upgrade_2_0.js:81]: Also forgotten to add the backend accounts as managers of the Factory and TokenPriceProvider

Comment (elenadimitrova) [reply deployment/7_upgrade_2_0.js:81]: And set the final owner of the WalletFactory to MultiSig also

File: deployment/7_upgrade_2_0.js [hunk 3]
```diff
@@ -1,23 +1,41 @@
...
+  "ApprovedTransfer",
+  "CompoundManager",
+  "GuardianManager",
+  "LockManager",
+  "NftTransfer",
+  "RecoveryManager",
+  "TokenExchanger",
+  "MakerV2Manager",
+  "TransferManager"];
+const MODULES_TO_DISABLE = [MakerManager];
```
Comment (juniset) [inline deployment/7_upgrade_2_0.js:38]: It should be ` ["MakerManager"] ` .

**Pull request merged**